core/services/keeper/delegate.go

```diff
@@ -53,6 +57,12 @@ func (d *Delegate) ServicesForSpec(spec job.Job) (services []job.Service, err er
 		return nil, errors.Wrap(err, "unable to create keeper registry contract wrapper")
 	}
 
+	// Make sure we can read the contract
```

Comment:
Would this cause the job to fail and stop attempting to execute if the ETH node is inaccessible briefly on startup?

Reply:
Yeah it would. I was thinking you could just delete and add, but thats not great. I'll move the spec_error creation to the GetConfig call in newRegistryFromChain and make it non-blocking